spec/horoshop_spec.rb

```diff
@@ -3,21 +3,89 @@
 require 'spec_helper'
 
 describe Horoshop do
+  subject { described_class.new(params) }
+
+  let(:url) { 'http://api.horosop.com' }
+  let(:username) { 'user213' }
+  let(:password) { 'pass' }
+  let(:params) do
+    { url: url, username: username, password: password }
```

Comment:
what you testing??? You are changed variable username to login in class Horoshop, but all tests are valid?

```
  let(:url) { 'http://api.horosop.com' }
  let(:username) { 'user213' }
  let(:password) { 'pass' }
  let(:params) do
    { url: url, username: username, password: password }
  end
```